apps/mobile-wallet/src/features/modals/modalActions.ts

```diff
@@ -4,6 +4,8 @@ import { ModalInstance, OpenModalParams } from '~/features/modals/modalTypes'
 
 export const openModal = createAction<OpenModalParams>('modal/openModal')
 
-export const closeModal = createAction<{ id: ModalInstance['id'] }>('modal/closeModal')
+export const closeModal = createAction<{ id?: ModalInstance['id']; name?: ModalInstance['params']['name'] }>(
+  'modal/closeModal'
+)
```

Comment:
With the suggestion I made above, this won't be necessary.

Reply:
Updated